app/views/content/funding-your-training.md

```diff
@@ -5,20 +5,12 @@
     school direct.
   date: "2021-03-11"
   image: "media/images/content/hero-images/0013.jpg"
-  subtitle: New events added - join an event near you and get all the info you need to become a teacher. 
-  subtitle_button: "Book your place"
-  subtitle_link: "/event-categories/train-to-teach-events"
+  subtitle: Get information and support to help you become a teacher. 
+  subtitle_button: "Register for updates"
+  subtitle_link: "/mailinglist/signup/name"
   backlink: "../"
   navigation: 25
   right_column:
-  ctas:
```

Comment:
Weirdly, I can't actually see this sidebar CTA in production (why is that @peteryates ?)

(If it was working) Do we need to remove this events CTA as well as changing the top promo?

Reply:
Ah the indentation isn't quite right there, `cts` should be inside `right_column`. Fixed it separately in #1779

Reply:
@gemmadallmandfe I would remove the events right hand CTA. Originally it was only there for the promotional space and (I _think_ )the stats show that it isn't used V much.

Reply:
I've looked in GA and it doesn't look well used so happy to remove